src/SSHDebugPS/StringResources.resx

```diff
@@ -172,16 +172,32 @@
     <value>The specified directory {0} could not be created or accessed.</value>
     <comment>{0} directory name.</comment>
   </data>
+  <data name="Error_OperationTimedOut" xml:space="preserve">
+    <value>Operation timed out.</value>
+  </data>
+  <data name="Error_ProcessListFailedFormat" xml:space="preserve">
+    <value>{0}
+
+{1}</value>
+    <comment>{0} = 'ps' error message, {1} = procfs error message</comment>
+  </data>
```

Comment:
Do we need some context text to indicate what the problem was? If not - no reason to localize this.

Reply:
I put it here to get the new lines to show up in the message dialog.

Reply:
I rewrote it